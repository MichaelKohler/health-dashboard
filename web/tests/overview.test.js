/* global it, expect */
/* eslint-disable react/jsx-filename-extension */

import React from 'react';
import { shallowToJson } from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Overview from '../src/overview.jsx';

it('renders correctly', () => {
  const wrapper = shallow(<Overview/>);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});

/* eslint-enable react/jsx-filename-extension */
