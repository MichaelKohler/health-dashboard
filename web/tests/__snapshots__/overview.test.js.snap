// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionAbsolute makeStyles-appBar-4 makeStyles-appBarShift-5 MuiAppBar-colorPrimary"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-2 MuiToolbar-gutters"
    >
      <button
        aria-label="open drawer"
        className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-6 makeStyles-menuButtonHidden-7 MuiIconButton-colorInherit MuiIconButton-edgeStart"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </span>
      </button>
      <h1
        className="MuiTypography-root makeStyles-title-8 MuiTypography-h6 MuiTypography-colorInherit MuiTypography-noWrap"
      >
        Dashboard
      </h1>
    </div>
  </header>
  <div
    className="MuiDrawer-root MuiDrawer-docked"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation0 MuiDrawer-paper makeStyles-drawerPaper-9 MuiDrawer-paperAnchorLeft MuiDrawer-paperAnchorDockedLeft"
    >
      <div
        className="makeStyles-toolbarIcon-3"
      >
        <button
          className="MuiButtonBase-root MuiIconButton-root"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
              />
            </svg>
          </span>
        </button>
      </div>
      <hr
        className="MuiDivider-root"
      />
      <ul
        className="MuiList-root MuiList-padding"
      >
        <div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemIcon-root"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 13h8V3H3v10zm0 8h8v-6H3v6zm10 0h8V11h-8v10zm0-18v6h8V3h-8z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
              >
                Dashboard
              </span>
            </div>
          </div>
          <hr
            className="MuiDivider-root"
          />
          <li
            className="MuiListSubheader-root MuiListSubheader-inset MuiListSubheader-sticky MuiListSubheader-gutters"
          >
            Reports
          </li>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemIcon-root"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
              >
                Cigarettes
              </span>
            </div>
          </div>
          <div
            aria-disabled={false}
            className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemIcon-root"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16 11c1.66 0 2.99-1.34 2.99-3S17.66 5 16 5c-1.66 0-3 1.34-3 3s1.34 3 3 3zm-8 0c1.66 0 2.99-1.34 2.99-3S9.66 5 8 5C6.34 5 5 6.34 5 8s1.34 3 3 3zm0 2c-2.33 0-7 1.17-7 3.5V19h14v-2.5c0-2.33-4.67-3.5-7-3.5zm8 0c-.29 0-.62.02-.97.05 1.16.84 1.97 1.97 1.97 3.45V19h6v-2.5c0-2.33-4.67-3.5-7-3.5z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
              >
                Weight
              </span>
            </div>
          </div>
        </div>
      </ul>
    </div>
  </div>
  <main
    className="makeStyles-content-12"
  >
    <div
      className="makeStyles-appBarSpacer-11"
    />
    <div
      className="MuiContainer-root makeStyles-container-13 MuiContainer-maxWidthLg"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-8 MuiGrid-grid-lg-9"
        >
          <div
            className="MuiPaper-root MuiPaper-elevation1 makeStyles-paper-14 makeStyles-fixedHeight-15 MuiPaper-rounded"
          >
            <h1>
              Hi!
            </h1>
          </div>
        </div>
      </div>
    </div>
    <p
      className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
    >
      Copyright Â© 2019 - 
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
        href="https://material-ui.com/"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        Michael Kohler
      </a>
       
    </p>
  </main>
</div>
`;
